packages/gasket-preset-api/package.json

```
@@ -1,12 +1,22 @@
 {
   "name": "@gasket/preset-api",
-  "version": "7.0.0-next.2",
+  "version": "7.0.0-next.20",
   "description": "Create Express-based API with Gasket",
-  "main": "lib",
+  "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "files": [
+    "generator",
     "lib"
   ],
+  "type": "module",
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "test": "cross-env NODE_OPTIONS='--unhandled-rejections=strict --experimental-vm-modules' jest",
+    "test:watch": "jest --watch",
```

Test watching is busted. Update to also use the node options:
```suggestion
    "test:watch": "npm run test -- --watch",
```